### DISCUSSION TITLE
How to collect email data in webhook?

### DISCUSSION THREAD

**Discussion by kayastha1988** [postalserver/postal#2478] [Help with using Postal]
In below image, it's list of email that I have sent via 3rd party email service provider.  I can see some emails are sent and some are bounced email. How to get those email data in webhook? In postal documentation I can see only the output but didn't get proper documentation for webhook.

Looking forward to a solution for this.

Thanks

![image](https://github.com/postalserver/postal/assets/19544006/abb890db-2ea5-4d6f-aefc-c9dc07945042)

**Comment by sinnbeck** [comment]
That is too late. You set up the webhooks **before** sending the emails and store/handle the data offsite. Now that they are already sent, you cannot sent the webhooks
